### DISCUSSION TITLE
Install deepmdkit-v2.1.5 with conda

### DISCUSSION THREAD

**Discussion by 13yong** [deepmodeling/deepmd-kit#2664] [Q&A]
I used conda to install deepmdkit-v2.1.5，It looks like succeed.

    $conda create -n dp deepmd-kit=2.1.5=*gpu libdeepmd=*=*gpu lammps cudatoolkit=10.2 horovod -c https://conda.deepmodeling.com -c defaults

The final screen output is
    
    Executing transaction: done
    To activate this environment, use
    $ conda activate dp
    To deactivate an active environment, use
    $ conda deactivate

But when I run

    dp -h

The error is as follows：
  
    WARNING:tensorflow:From /home/m-dpmd1/anaconda3/envs/dp/lib/python3.10/site- 
    packages/tensorflow/python/compat/v2_compat.py:107: disable_resource_variables (from tensorflow.python.ops.variable_scope) is 
    deprecated and will be removed in a future version.
    Instructions for updating:
    non-resource variables are not supported in the long term
    WARNING:root:To get the best performance, it is recommended to adjust the number of threads by setting the environment 
    variables OMP_NUM_THREADS, TF_INTRA_OP_PARALLELISM_THREADS, and TF_INTER_OP_PARALLELISM_THREADS.
    Traceback (most recent call last):
    File "/home/m-dpmd1/anaconda3/envs/dp/lib/python3.10/site-packages/deepmd/env.py", line 269, in get_module
       module = tf.load_op_library(str(module_file))
     File "/home/m-dpmd1/anaconda3/envs/dp/lib/python3.10/site-packages/tensorflow/python/framework/load_library.py", line 54, 
    in load_op_library
        lib_handle = py_tf.TF_LoadLibrary(library_filename)
     tensorflow.python.framework.errors_impl.NotFoundError: /home/m-dpmd1/anaconda3/envs/dp/lib/python3.10/site- 
     packages/deepmd/op/libop_abi.so: undefined symbol: 
    _ZN6deepmd33prod_env_mat_a_nvnmd_quantize_cpuIdEEvPT_S2_S2_PiPKS1_PKiRKNS_10InputNlistEiS5_S5_iiffSt6vectorIiSaIiEES5_

    The above exception was the direct cause of the following exception:

    Traceback (most recent call last):
      File "/home/m-dpmd1/anaconda3/envs/dp/bin/dp", line 7, in <module>
        from deepmd.entrypoints.main import main
      File "/home/m-dpmd1/anaconda3/envs/dp/lib/python3.10/site-packages/deepmd/__init__.py", line 7, in <module>
        import deepmd.utils.network as network
      File "/home/m-dpmd1/anaconda3/envs/dp/lib/python3.10/site-packages/deepmd/utils/__init__.py", line 2, in <module>
       from .data import DeepmdData
     File "/home/m-dpmd1/anaconda3/envs/dp/lib/python3.10/site-packages/deepmd/utils/data.py", line 10, in <module>
        from deepmd.env import GLOBAL_NP_FLOAT_PRECISION
     File "/home/m-dpmd1/anaconda3/envs/dp/lib/python3.10/site-packages/deepmd/env.py", line 354, in <module>
       op_module = get_module("op_abi")
     File "/home/m-dpmd1/anaconda3/envs/dp/lib/python3.10/site-packages/deepmd/env.py", line 325, in get_module
       raise RuntimeError(error_message) from e
      RuntimeError: This deepmd-kit package is inconsitent with TensorFlow Runtime, thus an error is raised when loading op_abi. You 
      need to rebuild deepmd-kit against this TensorFlow runtime.
      WARNING: devtoolset on RHEL6 and RHEL7 does not support _GLIBCXX_USE_CXX11_ABI=1. See 
     https://bugzilla.redhat.com/show_bug.cgi?id=1546704

So I hope to get your guidance.

**Comment by njzjz** [comment]
You installed `deepmd-kit` v2.1.5 but the latest version of `libdeepmd` in the same environment. Thus, the libraries will be conflicted.

**Reply by 13yong** [reply]
Firstly, thank you for your help.
So, what we need to do when meet this error?
And also I try some methods. Like #1905 , but I can't find the answer to resolve it. Finally I successfully install deepmdkit-v2.1.1 with 

    CONDA_OVERRIDE_CUDA=10.2 ~/anaconda3/bin/conda create -n dp-v211 deepmd-kit=*=*gpu -c https://conda.deepmodeling.org -y

But in this way, there is no **lmp** script.
The DeepModeling community has always started with user-friendliness, but the current experience is worse than before. I wonder if the developer can publish an installation tutorial including but not limited to deepmd-kit, lammps and dpgen.

**Reply by njzjz** [reply]
>  So, what we need to do when meet this error?

You need to install the same version as shown in the documentation.

The tutorial has been published at https://tutorials.deepmodeling.com/.

**Reply by njzjz** [reply]
cc @likefallwind
